bin/wmagent-mod-config

```diff
@@ -43,9 +43,23 @@ def importConfigTemplate(filename):
     Given filename, load it and grab the configuration object from it
 
     """
-    modSpecs = importlib.machinery.PathFinder().find_spec(filename)
-    mod = modSpecs.loader.load_module("wmcore_config_input")
-    config = getattr(mod, 'config', None)
+
+    cfgBaseName = os.path.basename(filename).replace(".py", "")
+    cfgDirName = os.path.dirname(filename)
+    if not cfgDirName:
```

Comment:
Todor, in which scenario a given filename would NOT have a dirname? I am trying to understand if we really need this if/else statement below.

Reply:
If one directly tries to load by module name and the search path is supposed to be the default system python search path, rather than determined by the relative path coming with the filename.

Reply:
If this method expects a filename as input, then it should not work if a module name is passed instead.
But now I understand when cfgDirName would be None, given that we strip the filename.